src/components/slide-transition/slide-transition.ts

```diff
@@ -0,0 +1,66 @@
+
+import { PluginObject } from 'vue';
+import Component from 'vue-class-component';
+import { Prop } from 'vue-property-decorator';
+import { ScrollToDuration } from '../../utils/scroll-to/scroll-to';
+import { ModulVue } from '../../utils/vue/vue';
+import { SLIDE_TRANSITION_NAME } from '../component-names';
+import WithRender from './slide-transition.html?style=./slide-transition.scss';
+
+@WithRender
+@Component
+export class MSlideTransition extends ModulVue {
+
+    @Prop({ default: true })
+    public leftToRight: boolean;
+
+    @Prop({ default: 0 })
+    public scrollToOffset: number; // the offset to add (in case of a sticky header)
+
+    @Prop({ default: false })
+    public disabled: boolean;
+
+    public get name(): string {
+        return this.disabled ? 'm--is-disabled' : 'm--is';
```

Comment:
La classe `'m--is-disabled'` devrait être directement appliquée sur `<div class="m-slide-transition">` et non sur `<transition :name="name" ...>`, car on ne veut pas appliquer la classe disabled sur l'item puisque c'est la transition qui est désactivée et non l'item.

Je propose donc cela: 
```
public get name(): string | undefined {
        return !this.disabled ? 'm--is' : undefined;
}
```
ou 

```
public get name(): string {
        return this.disabled ? '' : 'm--is';
}
```

Reply:
le getter name devrait être appellé transitionName

Reply:
Jai mis le undefined